packages/switchToggle/src/Default/Default.types.ts

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import * as bs from 'react-bootstrap';
+
+export interface SwitchToggleDefaultCommons {
+  theme?: 'Default' | 'Dark';
+  size?: 'Sm' | 'Md' | 'Lg';
+  prefix?: string;
+  children: React.ReactNode;
+}
+
+export type SwitchToggleDefaultProps = Partial<SwitchToggleDefaultCommons> &
```

Comment:
as all the `SwitchToggleDefaultCommons` are optional, we don't need the `Partial<>` here:
```ts
export type SwitchToggleDefaultProps = SwitchToggleDefaultCommons & bs.FormCheckProps;
```